pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>legalthings</groupId>
+  <artifactId>lto-api</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>lto-api</name>
+  <url>http://maven.apache.org</url>
+  
+	<repositories>
+		<repository>
+		    <id>jitpack.io</id>
+		    <url>https://jitpack.io</url>
+		</repository>
+		<repository>
+		    <id>dl.bintray.com</id>
+		    <url>https://dl.bintray.com/ethereum/maven/</url>
+		</repository>
+	</repositories>
+	
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>	
+  <dependencies>
+    <dependency>
+	    <groupId>junit</groupId>
+	    <artifactId>junit</artifactId>
+	    <version>4.12</version>
+	    <scope>test</scope>
+	</dependency>
+	<dependency>
+	    <groupId>org.apache.wink</groupId>
```

Comment:
I prefer to use jackson instead, because I had some trouble with Wink in the past. But is a personal preferences maybe you can argue why you want to use Wink?

Reply:
No special argue for this, that's from my own previous code.
If you don't agree with it, then let me know if I should replace it.